spec/services/middleware_topology_service_spec.rb

```diff
@@ -76,6 +99,28 @@
                                                                        :key          => "MiddlewareServer" + mw_server.compressed_id.to_s}
       )
 
+      expect(subject[:items]).to include(
+        "MiddlewareServerEap" + mw_server_eap.compressed_id.to_s   => {:name         => mw_server_eap.name,
+                                                                       :status       => "Running",
+                                                                       :kind         => "MiddlewareServerEap",
+                                                                       :display_kind => "MiddlewareServerEap",
+                                                                       :miq_id       => mw_server_eap.id,
+                                                                       :icon         => match(/vendor-jboss-eap/),
```

Comment:
is this the same as above?
https://github.com/ManageIQ/manageiq-ui-classic/pull/2724/files#diff-f32f59cd3f168fe8e6bb029d5b543f20R34
here it's vendor-jboss-eap and there it's just vendor-eap

Reply:
Yes, you are correct!